maps/atlas.dmm

```diff
@@ -2078,6 +2078,17 @@
 /obj/item/storage/box/ic_cones,
 /turf/simulated/floor/specialroom/freezer,
 /area/station/crew_quarters/catering)
+"ft" = (
+/obj/rack,
+/obj/item/breaching_charge,
+/obj/item/breaching_charge,
+/obj/item/breaching_charge,
+/obj/item/breaching_hammer{
```

Comment:
?

Reply:
Is it not normal for the additions and deletions to be shuffled around?